src/main/java/net/fabricmc/loom/util/srg/ForgeMappingsMerger.java

```diff
@@ -98,33 +101,34 @@ private SrgMerger(Path srg, Path tiny, @Nullable ExtraMappings extraMappings, bo
 			this.extra = null;
 		}
 
-		MappingReader.read(tiny, this.src);
-		checkInputNamespaces(tiny);
-
-		this.output.visitNamespaces(this.src.getSrcNamespace(), Stream.concat(Stream.of("srg"), this.src.getDstNamespaces().stream()).collect(Collectors.toList()));
+		this.output.visitNamespaces(this.src.getSrcNamespace(), Stream.concat(Stream.of(this.newNs.getDstNamespaces().get(0)), this.src.getDstNamespaces().stream()).collect(Collectors.toList()));
```

Comment:
The Stream.concat code here is honestly horrible, I'd rewrite it to just use an ArrayList